force-app/main/default/lwc/nksContactInformation/nksContactInformation.js

```diff
@@ -128,6 +133,10 @@ export default class NksContactInformation extends LightningElement {
     }
 
     populatePersonFields(data) {
+        this.personIdent = getFieldValue(data, NAME_FIELD);
+        if (this.personIdent !== '' && this.personIdent !== null) {
+            this.updateKrrInformation(this.personIdent);
+        }
```

Comment:
Her bør du også inkludere undefined i sjekken, ettersom personIdent ikke er initialisert og dermed er undefined i klassen.
`if (!!this.personIdent) `
eller
`if (this.personIdent !== '' && this.personIdent != null) {`